physics/mynnedmf_wrapper.F90

```diff
@@ -159,7 +161,8 @@ SUBROUTINE mynnedmf_wrapper_run(        &
      &  icloud_bl, do_mynnsfclay,                          &
      &  imp_physics, imp_physics_gfdl,                     &
      &  imp_physics_thompson, imp_physics_wsm6,            &
-     &  chem3d, frp, mix_chem, fire_turb, nchem, ndvel,    &
+     &  chem3d, frp, mix_chem, rrfs_smoke, fire_turb, nchem, ndvel, &
```

Comment:
same comment as in module_bl_mynn.F90, we shouldn't need both rrfs_smoke and mix_chem. The MYNN doesn't need to know the specific application, it only needs to know that the chem mixing routines are to be used.

Reply:
These changes are necessary to run on Cheyenne with the gnu compiler. If you want to try reducing the number of changes without causing the model to crash, they you're welcome to do so... if you have a Cheyenne account.

Reply:
Then there must be a bug in the smoke implementation part of the MYNN changes because it should only take one flag to activate/use that portion of the code. That was a definitely a rushed implementation that needs a revision.

Reply:
It was. I had an hour to get it working.